packages/design-system-tokens/src/themes/medicare-components.ts

```diff
@@ -120,13 +120,13 @@ export const components: AnyTokenValues = {
   },
 
   'button-solid': {
-    '__background-color--active':                 t.color['primary-darkest'],
+    '__background-color--active':                 t.color['primary-light'],
     '__background-color--disabled':               t.color['gray-lighter'],
-    '__background-color--hover':                  t.color['primary-darker'],
+    '__background-color--hover':                  t.color['primary-dark'],
     '__background-color':                         t.color['primary'],
-    '__border-color--active':                     t.color['primary-darkest'],
+    '__border-color--active':                     t.color['primary-light'],
     '__border-color--disabled':                   t.color['gray-lighter'],
-    '__border-color--hover':                      t.color['primary-darker'],
+    '__border-color--hover':                      t.color['primary-dark'],
```

Comment:
Why are these changes showing up here?

Reply:
Okay, I just tracked it down to https://github.com/CMSgov/design-system/pull/2001/commits/ee1d6ca7086b7ad37ca1dec0cba214c5746b8df8. @line47, can we remove this and add it to a separate pull request? It doesn't seem related to this change.